### PR TITLE
SEO-182846 ASP.NET Core MVC Page with Redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,
Have worked on On Page changes and Page with Redirect links
Regards,
Yvone.

### CODE REVIEW

File: ej2-asp-core-mvc/toast/EJ2_ASP.MVC/timeout.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 ---
 
 
-# Time out
+# Time out in Toast Control
```
Comment (gowthamanthavasiyappan) [inline ej2-asp-core-mvc/toast/EJ2_ASP.MVC/timeout.md:12]: Add platform name here

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: ej2-asp-core-mvc/toast/EJ2_ASP.MVC/timeout.md (hunk 1 above)
Comment (Yvone-Atieno) [reply ej2-asp-core-mvc/toast/EJ2_ASP.MVC/timeout.md:12]: We are using the same repo for Core and MVC so no need to specify the platform name

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Pull request merged**